index.html

```diff
@@ -25,21 +25,21 @@
                     <li><a href="#">Contact</a></li>
                 </ul>
                </div>
-        </div>
+        </nav>
     </header>
     <section class="section-1">
         <div class="section-1-container">
                 <h1 class="heading">I’m Mariane<br>Glad to see you!</h1>
                 <p class="leading">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
                 <p class="social-heading">LET’S CONNECT</p>
-                <div class="socials">
-                    <img src="images/Vector.svg">
-                    <img src="images/Linkedin icon.svg">
-                    <img src="images/Vector (2).svg">
-                    <img src="images/Vector (3).svg">
-                    <img src="images/Vector (4).svg">
-                </div>
+                <ul class="socials">
+                    <li><img src="images/Vector.svg"></li>
+                    <li><img src="images/Linkedin icon.svg"></li>
+                    <li><img src="images/Vector (2).svg"></li>
+                    <li><img src="images/Vector (3).svg"></li>
+                    <li><img src="images/Vector (4).svg"></li>
+                </ul>
```

Comment:
Great job implementing the changes mentioned by the previous reviewer, however, kindly use the `a` tag inside the `li` tag as this is semantically correct, and the `a ` tag will let you connect to other sections of the website, using href 👍 

>   
       <ul class="socials">
                    <li>< a href=""><img src="images/Linkedin icon.svg"> </a></li>
                       .
                       .
                       . 

       </ul>